recipes/openimageio/all/conandata.yml

```
@@ -0,0 +1,8 @@
+sources:
+  "2.2.6.1":
+    url: "https://github.com/OpenImageIO/oiio/archive/Release-2.2.6.1.tar.gz"
+    sha256: "adc245c9b2fa2bce1dd2decbdd1d03974e1e9818219d267d1da3dd1bd91216df"
+patches:
+  "2.2.6.1":
+    - patch_file: "patches/2.2.6.1.patch"
```

Maybe something more descriptive?

```suggestion
    - patch_file: "patches/2.2.6.1-cmake-targets-nowerror.patch"
```

---

Done.